sys/newlib/syscalls.c

```diff
@@ -39,6 +39,7 @@
 #include "reboot.h"
 #include "irq.h"
 #include "log.h"
+#include "vfs.h"
```

Comment:
This needs to be ```#ifdef MODULE_VFS```, otherwise I get this:

```
"make" -C /home/kaspar/src/riot.tmp/sys/newlib
In file included from /home/kaspar/src/riot.tmp/sys/newlib/syscalls.c:42:0:
/home/kaspar/src/riot.tmp/sys/include/vfs.h:59:50: fatal error: sys/statvfs.h: No such file or directory
 #include <sys/statvfs.h> /* for struct statvfs */
```